### DISCUSSION TITLE
How to make one user only see the data resources that he entered, so that other users cannot see it

### DISCUSSION THREAD

**Discussion by hagiik** [filamentphp/filament#15195] [Help]
### Package

Panel builder

### Package Version

V3

### How can we help you?

I have a blog resource, it contains the necessary data as a blog should, but I am confused about how to make only user A can see the data he wrote, but other users know what he wrote.

I am confused and can't find the tutorial, can you help me by giving an example or related article

Thank You

**Comment by leandrocfe** [comment]
Filament supports [Policies](https://filamentphp.com/docs/3.x/panels/resources/getting-started#authorization)
You can customize the [query or model scope](https://filamentphp.com/docs/3.x/panels/resources/getting-started#customizing-the-resource-eloquent-query) to affect the entire resource
